contracts/contracts/coordination/subscription/BqETHSubscription.sol

```diff
@@ -145,7 +150,14 @@ contract BqETHSubscription is EncryptorSlotsSubscription, Initializable, Ownable
     }
 
     function baseFees() public view returns (uint256) {
-        return baseFeeRate * subscriptionPeriodDuration * maxNodes;
+        uint256 currentPeriodNumber = getCurrentPeriodNumber();
+        return baseFees(currentPeriodNumber);
+    }
+
+    function baseFees(uint256 periodNumber) public view returns (uint256) {
+        uint256 baseFeeRate = initialBaseFeeRate *
+            (INCREASE_BASE + baseFeeRateIncrease * periodNumber);
+        return (baseFeeRate * subscriptionPeriodDuration * maxNodes) / INCREASE_BASE;
```

Comment:
This implementation doesn't factor in compounding interest (i.e., rate increases should be applied over the previous period base rate, not over the initial base rate). We could do something like this:
```suggestion
            (INCREASE_BASE + baseFeeRateIncrease) ** periodNumber;
        return (baseFeeRate * subscriptionPeriodDuration * maxNodes) / (INCREASE_BASE ** periodNumber);
```

Reply:
right! I'll change it

Reply:
✔️